include/aos/sm/resourcemanager.hpp

```diff
@@ -320,7 +320,7 @@ class ResourceManager : public ResourceManagerItf, private NonCopyable {
     static constexpr auto cConfigJSONLen = AOS_CONFIG_UNIT_CONFIG_JSON_LEN;
 
     Error LoadUnitConfig();
-    Error ValidateUnitConfig(const NodeUnitConfig& nodeUnitConfig) const;
+    Error ValidateUnitConfig(const NodeConfig& nodeConfig) const;
```

Comment:
`ValidateNodeConfig`?

Reply:
In this commit I've renamed NodeUnitConfig in the common
In the next commit I've renamed sm NodeConfig struct and updated method names.